agent/php_minit.c

```diff
@@ -722,6 +722,11 @@ PHP_MINIT_FUNCTION(newrelic) {
   nrl_debug(NRL_INIT, "MINIT processing done");
 #if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP 7.4+ */
   NR_PHP_PROCESS_GLOBALS(zend_offset) = zend_get_resource_handle(dummy);
+#if ZEND_MODULE_API_NO >= ZEND_8_4_X_API_NO /* PHP 7.4+ */
+  NR_PHP_PROCESS_GLOBALS(op_array_extension_handle) = zend_get_internal_function_extension_handle("newrelic");
```

Comment:
Why is `op_array_extension_handle` being set to the internal function extension handle for PHP 8.4+?

Reply:
This code no longer exists.